Alignment.go

```diff
@@ -20,6 +20,46 @@ const (
 	AlignRight
 )
 
+// AlignManually allows to apply alignment manually.
+// As long as AlignSetter is really EXPERIMENTAL feature
+// and may fail randomly, the following method is supposed to
+// always work, as long as you set it up correctly.
+// To use it just pass a single widget with its exact width.
+// be sure to apply widget's size by using "Size" method!
+// forceApplyWidth argument allows you to ask giu to force-set width
+// of `widget`
+// NOTE that forcing width doesn't work for each widget type! FOr example
+// Button won't work because its size is set by argument to imgui call
+// not PUshWidth api.
+func AlignManually(alignmentType AlignmentType, widget Widget, widgetW float32, forceApplyWidth bool) Widget {
```

Comment:
Suggest to use widgetWidth instead of widgetW as parameter's name.

Reply:
BTW, I think we don't need to pass the width as parameter here, because the size value is already be stored inside widget struct.

Reply:
probably, but this way this widget/method is more flexible and could be used in more situations.
e.g. for custom user's widgets or anything else, where user can know/calculate width.